core/modules/filter/js/filter.filtered_html.admin.js

```diff
@@ -56,9 +56,9 @@ Backdrop.behaviors.filterFilterHtmlUpdating = {
 
   attach: function (context, settings) {
     var that = this;
-    $(context).find('[name="filters[filter_html][settings][allowed_html]"]').once('filter-filter_html-updating').each(function () {
+    $(context).find('#allowed-html').once('filter-filter_html-updating').each(function () {
```

Comment:
some places use IDs and some use data attributes, is that ok?

Reply:
Yeah I think so. I used data attributes where a machine name was being embedded into the DOM element and IDs where we were hard-coding a particular string.